transaction-service/src/main/java/pl/adamboguszewski/transaction/service/application/TransactionService.java

```diff
@@ -59,13 +58,14 @@ public void deleteTransaction(Long id) {
         repository.deleteById(id);
     }
 
-    public List<Transaction> getAllTransactionsOlderThan2Years() {
-        LocalDate twoYearsAgo = LocalDate.now().minusYears(2);
-        return repository.findByTransactionInformation_TransactionDateTime_DateBefore(twoYearsAgo);
+    public List<Transaction> getAllOldTransactions() {
+        final int HOW_MANY_YEARS = 2;
```

Comment:
no no no no
do not define constant values like that. We will use properties later. For now just do the controller method, where you pass hard coded time in months to the service (deleteOldTransactions(Long months)).